### PR TITLE
Gainsight px entire event

### PR DESCRIPTION
**Pull request by nwolfe-xx**
_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/gainsight-px-cloud-action/index.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  {
+    name: 'Track Page View',
+    subscribe: 'type = "page"',
+    partnerAction: 'trackPageView',
+    mapping: defaultValues(trackPageView.fields)
+  }
+]
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Gainsight Px Cloud Action',
```
Comment (daltonscharff) [inline packages/destination-actions/src/destinations/gainsight-px-cloud-action/index.ts:41]:
```suggestion
  name: 'Gainsight Px Cloud (Actions)',
```
Adjusts name to fit naming convention

File: packages/destination-actions/src/destinations/gainsight-px-cloud-action/index.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  slug: 'gainsight-px-cloud-action',
```
Comment (daltonscharff) [inline packages/destination-actions/src/destinations/gainsight-px-cloud-action/index.ts:42]:
```suggestion
  slug: 'actions-gainsight-px-cloud',
```
Adjusts slug to fit naming convention

**Review by daltonscharff — CHANGES_REQUESTED** [review]
Looks great! I'll merge this after the slight changes are made to the name and slug.

**Comment by jinapark202** [comment]
@nwolfe-xx What's the difference between this PR and the one that was merged?

**Comment by jinapark202** [comment]
Closing because this PR seems to be a duplicate of PR #388

**Pull request closed without merging**